tasks/ansible.py

```diff
@@ -20,9 +20,15 @@ def _run(self):
 class AnsiblePlaybook(PopenTask):
     def __init__(self, playbook=None, extra_vars=None,
                  verbosity=None, **kwargs):
+        if self.extra_vars is None:
```

Comment:
```python
AttributeError: 'AnsiblePlaybook' object has no attribute 'extra_vars'
```
I'm changing this line to:
```python
if extra_vars is None:
```
However, only this method is using this object's attribute. Actually, the same applies to `self.verbosity` and `self.playbook`.